icu4c/packaging/distrelease.ps1

```diff
@@ -13,7 +13,7 @@
 #  (bring up Powershell ISE)
 #    cd C:\icu\icu4c\
 #    Set-ExecutionPolicy -Scope Process Unrestricted
-#    .\packaging\distrelease.ps1
+#    .\packaging\distrelease.ps1 -arch "x64 or x86"
```

Comment:
More of a nit, but I'd suggest saying "x64" or "x86" since the `or` isn't part of the option. ;)

Reply:
Hmm but the icucheck.bat and samplecheck.bat scripts also have the "x64 or x86" format.

Reply:
Ugh, that's unfortunate. I guess this is fine for now.